compiler/damlc/tests/daml-test-files/Interface.daml

```diff
@@ -7,9 +7,11 @@ module Interface where
 
 import DA.Assert ((===))
 
+data EmptyInterfaceView = EmptyInterfaceView
```

Comment:
This declares an enum, not a record, so it should be

```suggestion
data EmptyInterfaceView = EmptyInterfaceView {}
```

You can tell because the desugared version of this declaration has the ["stupid theta"](https://hackage.haskell.org/package/ghc-8.8.1/docs/DataCon.html#v:dataConStupidTheta) `GHC.Types.DamlEnum`:

```haskell
data GHC.Types.DamlEnum => EmptyInterfaceView = EmptyInterfaceView
```

Tangentially, I had a vague memory of the compiler failing with `data` declarations like this one, suggesting to make it explicitly a record with empty braces or explicitly a variant with empty parens. The docs do [mention this restriction](https://docs.daml.com/app-dev/daml-lf-translation.html#banned-declarations), but I wasn't able to replicate it even with `sdk-version: 1.14.0` - turns out [this restriction was removed in _**0**_.13.33 (2019-11-06)](https://blog.digitalasset.com/developers/release-notes/0.13.33)

Reply:
Digging a bit more, _that_ is the reason for the `GHC.Types.DamlEnum` stupid theta: it's only used for single-constructor enums to differentiate them from variants and records, while enums with more constructors don't need it, see https://github.com/digital-asset/daml/pull/3345

Reply:
Replaced all using `sed -i 's/data EmptyInterfaceView = EmptyInterfaceView/data EmptyInterfaceView = EmptyInterfaceView {}/' $(ag 'data EmptyInterfaceView = EmptyInterfaceView' -l)`, fingers crossed